pkg/cmd/controller/config/config.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	restclient "k8s.io/client-go/rest"
+)
+
+// Config is configuration struct for EtcdProxyController. It's used to wire CLI and the controller.
+type EtcdProxyController struct {
```

Comment:
I expected this directly in the controller package.

Reply:
Moved to the controller package, but I decided to leave the struct definitions in the `config.go` file. Is that okay?